.github/workflows/snackpub.yml

```diff
@@ -59,6 +59,9 @@ jobs:
       - name: 🚨 Lint snackpub
         run: yarn lint --max-warnings 0
 
+      - name: 🧪 Unit test website
+        run: yarn test --ci --maxWorkers 1
```

Comment:
Any reason --maxWorkers can't be higher? (Or just omit the option?)

Reply:
i just copied the --maxWorkers from other snack github jobs and don't know why actually. i think we don't have to keep this for snackpub github jobs. let's removed this.